Podfile.lock

```diff
@@ -10,4 +10,4 @@ SPEC CHECKSUMS:
   FBSnapshotTestCase: 3dc3899168747a0319c5278f5b3445c13a6532dd
   OCMock: a6a7dc0e3997fb9f35d99f72528698ebf60d64f2
 
-COCOAPODS: 0.38.2
+COCOAPODS: 0.39.0
```

Comment:
We should actually delete this file and the Pods directory, as it should be installed locally by each user with "pod install".  39.0 is actually not compatible with the Pinterest setup right now, so even if we leave the file, please remove this change.